package.json

```diff
@@ -14,7 +14,8 @@
   "scripts": {
     "start": "node src/app.js",
     "lint": "eslint .",
-    "test": "istanbul test -- _mocha",
+    "test": "istanbul test -- _mocha --timeout 150000",
```

Comment:
We should add mocha options to `test/mocha.opts` so that it uses the same options when you run `mocha` directly from the command line